src/OpenLoco/src/GameCommands/BuildCompanyHeadquarters.h

```diff
@@ -0,0 +1,36 @@
+#include "GameCommands.h"
```

Comment:
All these headers should have a pragma once

Reply:
Right, I forgot.